GPU/GPUCommon.cpp

```diff
@@ -1643,10 +1610,6 @@ void GPUCommon::Execute_Bezier(u32 op, u32 diff) {
 		indices = Memory::GetPointerUnchecked(gstate_c.indexAddr);
 	}
 
-	if ((gstate.vertType & GE_VTYPE_MORPHCOUNT_MASK) || vertTypeIsSkinningEnabled(gstate.vertType)) {
-		DEBUG_LOG_REPORT(G3D, "Unusual bezier/spline vtype: %08x, morph: %d, bones: %d", gstate.vertType, (gstate.vertType & GE_VTYPE_MORPHCOUNT_MASK) >> GE_VTYPE_MORPHCOUNT_SHIFT, vertTypeGetNumBoneWeights(gstate.vertType));
```

Comment:
Hmm, I'd argue that morph is still unusual for bezier/spline.  I think we should leave this and the above one..

-[Unknown]

Reply:
Well, the check was wrong before because vertTypeIsSkinningEnabled took bSoftwareSkinning into account. But yeah, I can agree with that.

Reply:
Though actually I think we handle this well. It's not really much stranger to have morphed and/or skinned vertices as an input to beziers/splines as regular draws. And now we know much more about all of this than when these debug logs were introduced. So I think I'll leave them out...

Reply:
Well, fair enough.

-[Unknown]